crypto/src/nullifier.rs

```diff
@@ -7,7 +7,7 @@ use once_cell::sync::Lazy;
 
 use crate::Fq;
 
-#[derive(PartialEq, Eq, Derivative, Clone, Hash, PartialOrd, Ord)]
+#[derive(PartialEq, Eq, Derivative, Clone, Copy, Hash, PartialOrd, Ord)]
```

Comment:
It's useful for `Nullifier` to be `Copy` and since other `Fq` newtype already are, I've done this here.